nux/store/reducer.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { combineReducers } from '@wordpress/data';
+
+export function guides( state = [], action ) {
```

Comment:
Would have liked to see JSDoc for each of these reducers to explain what they're tracking.

Reply:
👌 6af12cbd2